met-web/src/components/engagement/listing/index.tsx

```diff
@@ -265,9 +269,11 @@ const EngagementListing = () => {
                     </PrimaryButton>
                 </Stack>
             </Grid>
-            <When condition={advancedSearchOpen}>
-                <AdvancedSearch getFilterParams={filterParams} />
-            </When>
+            <Grid item xs={12} lg={10} style={{ width: '100%' }}>
+                <Collapse in={advancedSearchOpen} timeout="auto" unmountOnExit style={{ width: '100%' }}>
```

Comment:
should we unmountOnExit?  Not sure what is better. Perhaps if it was always mounted it will be faster and it's not a heavy component is it? Just for consideration

Reply:
updated to remove the same. thanks.